include/open62541/plugin/securitypolicy.h

```diff
@@ -320,6 +321,31 @@ struct UA_SecurityPolicy {
     void (*clear)(UA_SecurityPolicy *policy);
 };
 
+#ifdef UA_ENABLE_ENCRYPTION
+enum UA_CertificateFormat {
+    UA_CERTIFICATE_FORMAT_DER,
+    UA_CERTIFICATE_FORMAT_PEM
+};
+
+/**
+ * (Experimental) Create a certificate
+ *
+ * It is recommended to store the generated certificate in a non volatile memory and
+ * reuse it, so the application can be recognized across several executions.
+ *
+ * \param subject Elements for the subject,
+ *                  e.g. ["C=DE", "O=SampleOrganization", "CN=Open62541Server@localhost"]
+ * \param subjectAltName Elements for SubjectAltName,
+ *                  e.g. ["DNS:localhost", "URI:urn:open62541.server.application"]
+ */
+UA_StatusCode UA_EXPORT
+UA_CreateCertificate(const UA_Logger *logger,
```

Comment:
Keep consistent with the style of the overall SDK.
Array length are size_t.
The name of the array length is "arrayNameSize".
With a `Size` postfix.